src/main/java/taxi/controller/driver/AddDriverController.java

```diff
@@ -24,7 +24,9 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
     public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
         String name = req.getParameter("name");
         String licenseNumber = req.getParameter("license_number");
-        Driver driver = new Driver(name, licenseNumber);
+        String login = req.getParameter("login");
+        String password = req.getParameter("password");
+        Driver driver = new Driver(name, licenseNumber, login, password);
```

Comment:
we have RegistrationController, do we need this controller and vice versa?

Reply:
what do you mean, which controller ? RegistrationController or AddDriverController? if RegistrationController -yes, we need it, if AddDriverController -yes, we need it too